advtraj/cli/uclales.py

```diff
@@ -89,6 +90,10 @@ def sortkey(p):
         ds[c].attrs[f"d{c}"] = find_coord_grid_spacing(
             da_coord=ds[c], show_warnings=False
         )
+        if c in "xy":
```

Comment:
This is a bit cryptic to me, but I think this is adding the domain extent as tuples to the the attributes, right? Maybe you could add a comment indicating that? Or change the comment above to be `add the grid-spacing and domain extent...`

Reply:
I basically followed your excellent idea of including dx etc. to add Lx etc. I've modified the comment as suggested; see next commit,